src/App.jsx

```diff
@@ -0,0 +1,133 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+const Board = () =>  {
+  const [winner, setWinner] = useState(null);
+  const [board, setBoard] = useState(Array(9).fill(null));    //This creates and initializes the board
+  const [count, setCount] = useState(0);
+  const [squareColors, setSquareColors] = useState(Array(9).fill('blue')); 
+  
+
+  const winningCombos = [
+    [0, 1, 2], // Rows
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6], // Columns
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8], // Diagonals
+    [2, 4, 6],
+  ];
+
+ 
+
+  const checkWinner = (currentBoard) => {
```

Comment:
This looks good for this PR, no changes needed right now. But in a future PR do you think we could add support for the tied game functionality?

Reply:
For sure, let's fix it!